winpr/libwinpr/clipboard/clipboard.c

```diff
@@ -611,3 +611,145 @@ void ClipboardDestroy(wClipboard* clipboard)
 	DeleteCriticalSection(&(clipboard->lock));
 	free(clipboard);
 }
+
+static BOOL is_dos_driver(const char* path, size_t len)
```

Comment:
Typo? (should be `is_dos_drive` I guess)

Reply:
true.